### PR TITLE
doc issue

### PR DESCRIPTION
**Pull request by abishekrs24**

### CODE REVIEW

File: api/appengine.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ put game setup code and construct the view hierarchy.
 ### launchUI ()
 
 A user defined function that serves as the entry point for
-applications built on the Game Closure DevKit. This is executed
+applications built on the Game Closure Devkit. This is executed
```
Comment (rampr) [inline api/appengine.md]: Please put it back to DevKit with a capital 'K'

File: html/index.html [hunk 2]
```diff
@@ -318,21 +318,21 @@ <h2>Native Overview</h2>
...
-                  <a href="/guide/input.html">
+                  <a href="./guide/input.html">
                     <h2>Input Handling</h2>
                     <p>How to capture and make sense of input.</p>
                   </a>
                 </div>
 
-				<div class="span4">
-                  <a href="/guide/spriter.html">
+        <div class="span4">
```
Comment (rampr) [inline html/index.html:334]: indentation issue here

File: api/appengine.md (hunk 1 above)
Comment (rampr) [reply api/appengine.md]: Please change in all the places necessary

**Review by rampr — APPROVED** [review]

**Comment by rampr** [comment]
Write meaningful commit messages. You should be able to go back and understand what change was made from the commit message itself.

**Review by codesandeep — APPROVED** [review]

**Review by vivekb91 — APPROVED** [review]

File: build/template.html [hunk 3]
```diff
@@ -268,7 +268,7 @@ <h3 id="help-modal-label">Keyboard Shortcuts</h3>
       if (typeof(Zenbox) !== "undefined") {
         Zenbox.init({
           dropboxID:   "20148148",
-          url:         "https://gcdevkit.zendesk.com",
+          url:         "https://gcgameclosure.zendesk.com",
```
Comment (bijoshtj) [inline build/template.html:271]: does this exists?

Comment (rampr) [reply build/template.html:271]: it doesn't we can ignore these :)

**Pull request merged**